pkg/spdx/builder.go

```diff
@@ -241,20 +241,43 @@ func (builder *defaultDocBuilderImpl) GenerateDoc(
 	}
 
 	// Process single files, not part of a package
-	for _, f := range genopts.Files {
-		logrus.Infof("Processing file %s", f)
-		f, err := spdx.FileFromPath(f)
+	for _, filePattern := range genopts.Files {
+		matches, err := filepath.Glob(filePattern)
 		if err != nil {
-			return nil, errors.Wrap(err, "adding file")
+			return nil, err
 		}
-		doc.ensureUniqueElementID(f)
-		if err := doc.AddFile(f); err != nil {
-			return nil, errors.Wrap(err, "adding file to document")
+		if len(matches) == 0 {
+			logrus.Warnf("%s pattern didn't match any file", filePattern)
+		}
+		for _, filePath := range matches {
+			isFile, err := pathIsOfFile(filePath)
+			if err != nil {
+				return nil, errors.Wrap(err, "stat file")
+			}
+			if !isFile {
+				continue
+			}
+			f, err := spdx.FileFromPath(filePath)
+			if err != nil {
+				return nil, errors.Wrap(err, "adding file")
+			}
+			doc.ensureUniqueElementID(f)
+			if err := doc.AddFile(f); err != nil {
+				return nil, errors.Wrap(err, "adding file to document")
+			}
 		}
 	}
 	return doc, nil
 }
 
+func pathIsOfFile(path string) (bool, error) {
```

Comment:
Non blocking: At some point we would want to move this logic to live along this function:

https://github.com/kubernetes-sigs/release-utils/blob/71e6d370ca0be5ad5a9d5ed92eb4156513f154df/util/common.go#L484

Reply:
I left a commented `TODO` for this, hope that's enough :)